### PR TITLE
feat: add docs for enhanced user metrics

### PR DESCRIPTION
**Pull request by bpmct**
We shouldn't merge this until v1.34 (I didn't see a branch)

### CODE REVIEW

File: guides/api.md [hunk 1]
```diff
@@ -69,7 +84,7 @@ curl --request GET \
 limit=10\
 &resource_id=$WS_ID_PHP\
 &resource_type=environment" \
-  --header "Session-Token: $API_KEY" 
+  --header "Session-Token: $API_KEY"
```
Comment (bpmct) [inline guides/api.md:87]: It looks like the linter picked up some formatting things

File: comparison.md [hunk 2]
```diff
@@ -77,6 +77,6 @@ support.
 > To get a free trial of Coder, please visit
 > [https://coder.com/trial](https://coder.com/trial).
 
-> Coder's trial license does not work in an air-gapped environment. If your
-> organization is interested in evaluating Coder air-gapped, please contact
-> [sales@coder.com](mailto:sales@coder.com) to discuss license requirements.
+Coder's trial license does not work in an air-gapped environment. If your
+organization is interested in evaluating Coder air-gapped, please contact
+[sales@coder.com](mailto:sales@coder.com) to discuss license requirements.
```
Comment (bpmct) [inline comparison.md:82]: 0cffb5081daf30820f041d94ebf49782f22362ae introduced a linter warning that caused tests to fail

File: admin/metrics.md [hunk 3]
```diff
@@ -7,3 +7,119 @@ Metrics allow you to track the number of people using Coder on a day-to-day
...
+
+  - **Any**: Any of the below actions count towards activity
+
+  - **IDE**: Used a web IDE in a workspace (Code Web, JetBrains, PyCharm,
+    Jupyter, RStudio)
+
+  - **Login**: Logged in to Coder
+
+  - **App**: Used a custom
+    [workspace application](../workspaces/applications.md) in a workspace
```
Comment (Emyrk) [inline admin/metrics.md:29]: I wonder if we should somehow mention it only tracks long lived requests such as websockets. Short lived connections will appear only as 1 "event"

File: admin/metrics.md [hunk 4]
```diff
@@ -7,3 +7,119 @@ Metrics allow you to track the number of people using Coder on a day-to-day
...
+            "total_count": 3708,
+            "total_duration": 360000000000 // 61h 48m
+        }
+        "5f905429-ba6e4ac480eb4c0ead160b47": {
+            "id": "5f905429-ba6e4ac480eb4c0ead160b47",
+            "name": "Mark Milligan",
+            "username": "mark",
+            "roles": [
+                "site-member"
+            ],
```
Comment (Emyrk) [inline admin/metrics.md:118]: Can we fake these names?

`Alice`, `Bob`, `Charlie` etc?

File: guides/api.md [hunk 5]
```diff
@@ -58,7 +58,22 @@ curl --request GET \
   --url "$ACCESS_URL/$API_ROUTE/metrics/usage/active-users?\
```
Comment (Emyrk) [inline guides/api.md:58]: I wonder if we should still promote this endpoint. It does something similar :man_shrugging:

**Review by Emyrk — APPROVED** [review]

File: admin/metrics.md (hunk 4 above)
Comment (bpmct) [reply admin/metrics.md:118]: sure :)

**Comment by ericpaulsen** [comment]
closes [this issue](https://github.com/coder/v1/issues/12887).

File: guides/api.md (hunk 5 above)
Comment (bpmct) [reply guides/api.md:58]: We can remove it :)

**Pull request merged**